src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -55,9 +55,10 @@
         %li.nav-item.scrollable-tab-link#changes-item{ 'data-request-number': @bs_request.number, 'data-request-action-id': @actions.first[:id] }
           = link_to('Changes', '#changes', class: 'nav-link text-nowrap', 'aria-controls': 'changes', 'aria-selected': 'false',
                     'data-toggle': 'tab', role: 'tab')
-      %li.nav-item.scrollable-tab-link
-        = link_to('Mentioned Issues', '#mentioned-issues', class: 'nav-link text-nowrap', 'aria-controls': 'mentioned-issues',
-                  'aria-selected': 'false', 'data-toggle': 'tab', role: 'tab')
+      - if @action[:type].in?(actions_for_diff)
```

Comment:
This check should also be done below where we render the tab's content.

Reply:
True! done